<launch>
  <node name="create_grasp_database" pkg="chair_manipulation_grasp_detection_advanced" type="create_grasp_database_node" output="screen">
    <rosparam file="$(find chair_manipulation_grasp_detection_advanced)/cfg/pipeline.yaml" />
    <param name="gripper_urdf" command="cat $(find chair_manipulation_grasp_detection_advanced)/cfg/gripper/robotiq_arg2f_140.urdf" />
    <param name="gripper_srdf" command="cat $(find chair_manipulation_grasp_detection_advanced)/cfg/gripper/robotiq_arg2f_140.srdf" />
  </node>
</launch>